integration-tests/oidc-wiremock-providers/src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+quarkus.http.auth.proactive=false
+quarkus.log.category."org.htmlunit".level=ERROR
+quarkus.log.category."com.github".level=ERROR
+quarkus.keycloak.devservices.enabled=false
+
+quarkus.oidc.slack.provider=slack
+quarkus.oidc.slack.auth-server-url=${keycloak.url}/slack
+quarkus.oidc.slack.client-id=7925551513107.7922794171477
+quarkus.oidc.slack.credentials.secret=2b82d6039bc97946460fdec75fadd9b2
+quarkus.oidc.slack.authentication.extra-params.team=quarkus-oidc-slack-demo
+quarkus.oidc.slack.tenant-paths=/providers/slack*
```

Comment:
Propose to use the annotation, or if the the root resource path is `/slack` then if test calls `GET /slack` the default resolver will do... The simpler the test properties setup we can get the better